packages/core/src/routes/connector.ts

```diff
@@ -134,9 +135,12 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
       );
 
       const insertConnectorId = generateConnectorId();
+      const { metadata, ...rest } = body;
+
       ctx.body = await insertConnector({
         id: insertConnectorId,
-        ...body,
+        ...(metadata && { metadata: cleanDeep(metadata) }),
+        ...rest,
```

Comment:
Do we need use `transpileLogtoConnector` to construct the `ConnectorResponse`?

Reply:
In these standard connector PRs, we do not change the response type of all APIs since this could break those use cases rely on these APIs. The response of the connector APIs will be checked and aligned after we finish this standard connector feature.

Reply:
https://linear.app/silverhand/issue/LOG-4857/check-return-type-of-post-connector-api

Reply:
https://linear.app/silverhand/issue/LOG-4857/check-return-type-of-post-connector-api